### PR TITLE
Redesigned information card

### PR DESCRIPTION
**Pull request by shadab-skhan**
Redesigned information card
Used backend generated colors for discrete logs

### CODE REVIEW

File: react/src/lib/components/DeckGLMap/layers/utils/propertyMapTools.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import { PickInfo } from "@deck.gl/core/lib/deck";
 import { RGBColor } from "@deck.gl/core/utils/color";
+import { LayerPickInfo } from "../../components/InfoCard";
```
Comment (alexandruandrei91) [inline react/src/lib/components/DeckGLMap/layers/utils/propertyMapTools.ts:2]: In the future, the map component could be packaged by itself, so we shouldn't have dependencies on other internal components unless they are really used in the map.

You can move the needed classes (LayerPickInfo and PropertyDataType) into a file in layers/utils.

File: react/src/lib/components/DeckGLMap/layers/utils/propertyMapTools.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 
-export interface BitmapPickInfo extends PickInfo<unknown> {
+export interface BitmapPickInfo extends LayerPickInfo {
```
Comment (alexandruandrei91) [inline react/src/lib/components/DeckGLMap/layers/utils/propertyMapTools.ts:4]: Nothing from the LayerPickInfo is being filled in in the bitmap layer, so probably it doesn't need to extend it.
From the current usecase, LayerPickInfo includes information needed only for the Wells layer, so you can maybe add the fields to the WellPickInfo struct directly.

File: react/src/lib/components/DeckGLMap/components/InfoCard.tsx [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+import { Button, Icon } from "@equinor/eds-core-react";
+import { arrow_drop_up, arrow_drop_down } from "@equinor/eds-icons";
+
+import { PickInfo } from "deck.gl";
+import { PropertyMapPickInfo } from "../layers/utils/propertyMapTools";
+
+Icon.add({ arrow_drop_up, arrow_drop_down });
+
+export interface LayerPickInfo extends PickInfo<unknown> {
+    properties?: PropertyDataType[];
```
Comment (alexandruandrei91) [inline react/src/lib/components/DeckGLMap/components/InfoCard.tsx:20]: Does this have to be an array? Afaik, the multipicking method already returns an array of PickInfo objects.

File: react/src/lib/components/DeckGLMap/layers/utils/propertyMapTools.ts (hunk 2 above)
Comment (shadab-skhan) [reply react/src/lib/components/DeckGLMap/layers/utils/propertyMapTools.ts:4]: I planned to keep the common PickInfo type for all the layers required by the new info card.
But true, from the current use case only the wells layer needs this.

**Review by alexandruandrei91 — APPROVED** [review]
👍

**Pull request merged**